src/AccessControl/User.py

```diff
@@ -35,6 +35,9 @@
 from AccessControl.users import system
 
 
+from AccessControl.users import UnrestrictedUser # noqa isort:skip
```

Comment:
What is the reason for `isort:skip` ?

Reply:
To fix isort check and travis build : https://travis-ci.org/zopefoundation/AccessControl/jobs/637872954?utm_medium=notification&utm_source=github_status

Reply:
If I understand well, `isort` does not like `#noqa`. I'd add a comment explaining this.

Reply:
isort does not use `noqa` comment, if `isort:skip` is not specified the import is removed by isort.

`noqa` is for Flake8 / Pep8